.build/parent-pom-template.xml

```diff
@@ -675,14 +675,14 @@
         <version>0.4</version>
       </dependency>
       <dependency>
-        <groupId>accord</groupId>
-        <artifactId>accord</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <groupId>org.apache.cassandra</groupId>
+        <artifactId>cassandra-accord</artifactId>
+        <version>@version@</version>
       </dependency>
       <dependency>
-        <groupId>accord</groupId>
-        <artifactId>accord</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <groupId>org.apache.cassandra</groupId>
+        <artifactId>cassandra-accord</artifactId>
+        <version>@version@</version>
```

Comment:
So for now we'll just keep this in lock-step w/ the C* version?

Reply:
correct, as this is the cassandra release.  We can migrate off building and use an already published jar later on if we want.

The one negative to this is that if we migrate to the jar, the group/artifact/versions are different, so this "may" impact users of -all jar.